storybook/src/mock/documents.mock.js

```diff
@@ -0,0 +1,192 @@
+import image from './nuxeo-logo.jpg';
+
+export const DOCUMENT1 = {
+  'entity-type': 'document',
+  uid: '1',
```

Comment:
nitpick: we should have an id with a valid uuid format here.

Reply:
Indeed.
Anyway, since the UID is the only thing that changes from one document to the other, we may want to export a method that generates this kind of dummy documents, instead of having it replicated X times.

Reply:
will do it

Reply:
We can use the uuid lib that we're already [using on Web UI](https://github.com/nuxeo/nuxeo-web-ui/blob/master/package.json#L149).